firedrake/mg/mesh.py

```diff
@@ -9,6 +9,12 @@
 from firedrake.cython import mgimpl as impl
 from .utils import set_level
 
+try:
+    import netgen
+    import ngsPETSc
```

Comment:
Do we even need to import `ngsPETSc` here? We don't seem to use it. There is only a
```py
from ngsPETSc import NetgenHierarchy
```
below.